src/js/components/Anchor/Anchor.js

```diff
@@ -11,77 +11,70 @@ import { withFocus, withForwardRef } from '../hocs';
 
 import { StyledAnchor } from './StyledAnchor';
 
-class Anchor extends Component {
-  constructor(props) {
-    super(props);
-
-    const { children, icon, label } = props;
+const Anchor = ({
+  a11yTitle,
+  children,
+  color,
+  disabled,
+  forwardRef,
+  href,
+  icon,
+  focus,
+  label,
+  onClick,
+  reverse,
+  theme,
+  ...rest
+}) => {
+  useEffect(() => {
```

Comment:
hm, not sure we need a useEffect hook here. you can just add the test without it, if prop is changing then the component will re-render anyway.

Reply:
I used useEffect here because this logic was in constructor, with useEffect I guarantee that will exec just if these props been changed

Reply:
ok fine you are also avoiding to check if state changes. fine by me